cmd/snap/cmd_services.go

```diff
@@ -144,7 +149,7 @@ func (s *svcStatus) Execute(args []string) error {
 		if svc.Active {
 			current = i18n.G("active")
```

Comment:
as you mentioned current doesn't make sense for user units listed at this level,  we might want to set it to "-"?